Gemfile

```diff
@@ -40,6 +40,8 @@ gem "email_validator", "~> 1.6"
 # HTTP library wrapper
 gem "faraday", "~> 0.9.2", require: false
 
+gem "font-awesome-rails"
```

Comment:
Should we add 
```
= require font-awesome
```
in our .css? and ensure that it works?

Reply:
Yeah I forgot to commit it when I was testing.

Reply:
Can we lock down this dependency to a specific version?